.travis.yml

```diff
@@ -1,14 +1,37 @@
-language: go
-go:  
-   - "1.8.x"
-   - "1.9.x"
-   - "1.10.x"
-   - "1.11.x"
-before_script: 
-    - export GOPATH="$HOME/gopath"
-    - go get -d -v
-    - make prepare
-    - "export LD_LIBRARY_PATH=$PWD/libgit2/install/lib"
-    - "export PKG_CONFIG_PATH=$PWD/libgit2/install/lib/pkgconfig"
-script: 
-   - make test
```

Comment:
Did you either remove accidentally the running tests or it was on purpose?

Reply:
It's unintentional. I'll add this later

Reply:
I made some changes to `Makefile`, `static_build.sh` and `.travis.yml` in this commit: https://github.com/cloudson/gitql/pull/82/commits/8f9dda852fe02cb209c7052f9f5dc5efa0aa3b7a

Now it is possible to run `make test` and `make build` on both linux/amd64 and darwin/amd64 host environment. Please check it out :)

Tested with go1.13beta1 and go1.12.7 without any problem

And cross compiling can be done through passing the TARGET_OS_ARCH variable to make: `make TARGET_OS_ARCH=windows/amd64 build` (only tested inside linux/amd64 docker container, otherwise you would have to manually install all the cross compiling toolchain, which IMO is not worth the effort. Here is the Dockerfile for the navigaid/osxcross docker image: https://github.com/navigaid/osxcross-extras/blob/master/osxcross-builder/Dockerfile)